### PR TITLE
AnkitaM_Exam_1_30min

### PR DESCRIPTION
**Pull request by AnkitaDOct2020**
--Added test1 program.
image

### CODE REVIEW

File: src/ankita/FindMaximumWord.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package ankita;
+
+public class FindMaximumWord {
+	String findMaxWord(String[] arr) {
+		int max =0;
+		String maxWord=arr[0];
+		for(int index=0;index<arr.length;index++) {
+			String name = arr[index];
+			int cnt=0;
+			for(int i =0;i<name.length();i++) {
```
Comment (MaulikTechnocredits) [inline src/ankita/FindMaximumWord.java:26]: why do we need loop ????
can we directly say cnt= name.length() ???? even we dont need cnt.
dont create unnecessary vairables

File: src/ankita/FindMaximumWord.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+				cnt = cnt +1;
+			}
+			if(max<cnt) {
```
Comment (MaulikTechnocredits) [inline src/ankita/FindMaximumWord.java:29]: what if we remove max variable and write it like below.
if(maxWord.length()<arr[index].length())

**Review by MaulikTechnocredits — CHANGES_REQUESTED** [review]
Please work on comments and push it again.

File: src/ankita/FindMaximumWord.java (hunk 2 above)
Comment (AnkitaDOct2020) [reply src/ankita/FindMaximumWord.java]: Code corrected.

File: src/ankita/FindMaximumWord.java (hunk 1 above)
Comment (AnkitaDOct2020) [reply src/ankita/FindMaximumWord.java]: Corrected.

**Review by MaulikTechnocredits — APPROVED** [review]
Excellent job Ankita. Now everything looks as expected.

**Pull request merged**